CIMgen.py

```diff
@@ -328,7 +408,7 @@ def _parse_rdf(input_dic):
         else:
             logger.info("Class {} for instance {} not found.".format(clarse, instance))
 
-    return {package_name: classes_map}
+    return {"class_name": classes_map}
```

Comment:
Hi @guillaume-alliander, I believe this change caused an issue with the backwards compatibility of the code. I generated the Python output of CGMES Version 2.4.15 with the code in the master branch and in this branch and diffed the results. The resulting Python classes are using the "class_name" placeholder instead of the CGMES Profile strings "EQ", "DY", "TP_BD", "EQ_BD" etc. Can you please take a look?

Reply:
I expect that it should now be OK, tested on V2 and V3. 
(note: I rebased this branch on the current master).